bindata/network/openshift-sdn/sdn.yaml

```diff
@@ -174,6 +174,70 @@ spec:
             command: ["test", "-f", "/etc/cni/net.d/80-openshift-network.conf"]
           initialDelaySeconds: 5
           periodSeconds: 5
+      - name: kube-rbac-proxy
+        image: {{.KubeRBACProxyImage}}
+      - command:
+        - /bin/bash
+        - -c
+        - |
+          #!/bin/bash
+          set -euo pipefail
+          TLS_PK=/etc/pki/tls/metrics-certs/tls.key
+          TLS_CERT=/etc/pki/tls/metrics-certs/tls.crt
+
+          # As the secret mount is optional we must wait for the files to be present.
+          # The service is created in monitor.yaml and this is created in sdn.yaml.
+          # If it isn't created there is probably an issue so we want to crashloop.
+          TS=$(curl \
+            --cacert /var/run/secrets/kubernetes.io/serviceaccount/ca.crt \
+            -H "Authorization: Bearer $(cat /var/run/secrets/kubernetes.io/serviceaccount/token)" \
+            "https://${KUBERNETES_SERVICE_HOST}:${KUBERNETES_SERVICE_PORT}/api/v1/namespaces/openshift-sdn/services/sdn-metrics" |
+              python -c 'import json,sys; print(json.load(sys.stdin)["metadata"]["creationTimestamp"])'
```

Comment:
Can you just use `jq` here?

Reply:
No, the image doesn't have jq, kubectl or oc.

The image ships the urlgrabber python library so I probably could replace bash with a python script doing something like:
```
command:
  - python
  - -c
  - |
    <SNIP>
```

Not quite sure it's worth the effort now.